module_main.tf

```diff
@@ -52,8 +54,15 @@ module "elastic_beanstalk_environment" {
   description                = var.description
   region                     = var.region
   availability_zone_selector = var.availability_zone_selector
+  # NOTE: We would prefer for the DNS name 
+  # of module.elastic_beanstalk_environment
+  # to be staticly set via inputs,
+  # but have been running into other/different problems
+  # trying to get that to work 
+  # (for one thing, permissions error anytime try to set
+  # elastic_beanstalk_environment.dns_zone_id)
+  # dns_zone_id                = data.aws_elastic_beanstalk_hosted_zone.current.id
```

Comment:
problem #2 is that setting `dns_zone_id` triggers a permissions error, and docs for this module say we need to set the `dns_zone_id` as a prerequisite for setting a static DNS name for this beanstalk environment (need to double check and see if we can set the beanstalk DNS name without setting zone id)